### ISSUE TITLE
Collaboration with node-based @bazel/create

### ISSUE THREAD

**Issue by alexeagle** [jin/create-bazel-workspace#1]
You create a WORKSPACE file
https://github.com/jin/create-bazel-workspace/blob/92c1ab3c1ba87a811653ed4a108fc67959468dfa/create-bazel-workspace.go#L65
I create a WORKSPACE file
https://github.com/bazelbuild/rules_nodejs/blob/0c165b75d7b4a619f84cba139f922f123178d1df/packages/create/index.js#L174

We each would like to have a tool that makes a polyglot workspace with lots of languages supported which is ambitious like Gazelle in the need for many plugins

Seems like we should have just one tool (write it in one language) then publish it into multiple ecosystems so I can use whatever binary in my $PATH to run it, like
- `go install bazel-create && $GO_PATH...`
- `npx @bazel/create` 
- `mvn archetype:generate -DarchetypeArtifactId=maven-archetype-quickstart`
- all the others...

**Comment by jin** [comment]
I agree and I'd love to help, but I don't have the bandwidth to work on this now. Also we should align with bazel-federation seems we all seem to be solving the package management problem.

A `bazel init` / project bootstrapping feature will also fit nicely into `bazelisk` too, IMO, so that can be the "one tool".
